spec/system/admin/instrument_price_policies_controller_spec.rb

```diff
@@ -358,23 +357,32 @@
       expect(page).to have_field("price_policy_#{base_price_group.id}[duration_rates_attributes][1][rate]", with: "40.00")
       expect(page).to have_field("price_policy_#{base_price_group.id}[duration_rates_attributes][2][rate]", with: "30.00")
 
-      fill_in "min_duration_2", with: "5"
-      fill_in "price_policy_#{base_price_group.id}[duration_rates_attributes][2][rate]", with: "20"
+      fill_in "min_duration_1", with: "5"
+      fill_in "price_policy_#{base_price_group.id}[duration_rates_attributes][1][rate]", with: "20"
+
+      fill_in "min_duration_2", with: ""
+      fill_in "price_policy_#{base_price_group.id}[duration_rates_attributes][2][rate]", with: ""
+      fill_in "price_policy_#{cancer_center.id}[duration_rates_attributes][2][subsidy]", with: ""
+      fill_in "price_policy_#{external_price_group.id}[duration_rates_attributes][2][rate]", with: ""
+
+      find_field("price_policy_#{base_price_group.id}[duration_rates_attributes][1][rate]").native.send_keys :tab # change focus to trigger the onchange event
 
-      find_field("price_policy_#{base_price_group.id}[duration_rates_attributes][2][rate]").native.send_keys :tab # change focus to trigger the onchange event
-      expect(page).to have_field("price_policy_#{cancer_center.id}[duration_rates_attributes][2][rate]", with: "20", type: :hidden)
-      expect(page).to have_field("price_policy_#{external_price_group.id}[duration_rates_attributes][2][rate]", with: "90.00")
+      expect(page).to have_field("price_policy_#{cancer_center.id}[duration_rates_attributes][1][rate]", with: "20", type: :hidden)
+      expect(page).to have_field("price_policy_#{external_price_group.id}[duration_rates_attributes][1][rate]", with: "100.00")
 
       click_button "Save Rules"
 
       expect(page).to have_content("Price Rules were successfully updated.")
 
       expect(page).to have_content("Over 5 hrs")
+      expect(page).not_to have_content("Over 3 hrs")
+
       expect(page).not_to have_content("Over 4 hrs")
+      expect(page).not_to have_content("$40.00")
```

Comment:
I'd group the `not_to` expectations together with a code comment explaining the intent

Reply:
I grouped them by what they're testing and added comments